tests/foreman/ui/test_bookmarks.py

```diff
@@ -40,15 +94,21 @@ def test_positive_create_bookmark_populate_auto(self):
 
         @Assert: No errors, Bookmark is displayed, controller matches the
         entity the bookmark was created for
-
-        @Status: Manual
         """
-        for entity in BOOKMARK_ENTITIES:
-            if entity.get('setup'):
-                create_entity(entity['setup'])
-            pass
+        for entity in self.entities:
+            with self.subTest(entity):
+                with Session(self.browser) as session:
```

Comment:
Opening a new instance will allow some isolation between each entity. This is the same that having on test method for every single entity. I think in this case we can open a new session for each one considering the isolation it will provide even though we are adding overhead since it have to do the login and logout for every entity.